setup.cfg

```diff
@@ -27,8 +27,6 @@ install_requires =
     numpy>=1.14.5
     tox>=3.1.2
 
-[options.package_data]
-edgePy = data/*.csv*, data/*.tsv*, data/*.txt*
```

Comment:
By using these `setup.cfg` lines we get rid of the need for a `MANIFEST.in` and keep clutter down. Everything was packaging well on install before, why change?

Reply:
It wasn't packaging at all before - but if you can get it to work (I couldn't), I'd be happy to switch it back.

Reply:
The longer explanation is that the egg packaging seems to be determining what gets pulled along in the data set - this doesn't do anything that I'm aware of.

Instead, the manifest.in file causes the egg to pull the appropriate files in, which are then found by tox.  If I've done something wrong, please let me know.